sonar-plugin-api/src/main/java/org/sonar/api/batch/rule/Checks.java

```diff
@@ -78,7 +78,9 @@
  * It replaces org.sonar.api.checks.AnnotationCheckFactory
  *
  * @since 4.2
+ * @deprecated since 6.0 sonar-check-api is deprecated
  */
+@Deprecated
 public class Checks<C> {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)